h/tasks/mailer.py

```diff
@@ -19,23 +23,30 @@
 )
 def send(  # noqa: PLR0913
     self,  # noqa: ARG001
-    recipients: list[str],
-    subject: str,
-    body: str,
-    tag: EmailTag,
+    recipients: list[str] | dict[str, Any],
```

Comment:
Ouch, the backwards-compat support is quite confusing! But I know you've got a follow-up PR to change this anyway